data/docs/operate/migration/upgrade-0.55.mdx

```diff
@@ -134,3 +139,155 @@ Flags:
+  docker exec -it signoz-clickhouse /bin/bash
+  
+  clickhouse client
+  ```
+* Get the starting timestamp 
+  ```
+  select timestamp from signoz_logs.distributed_logs order by timestamp asc limit 1
+  ```
+* Get the ending timestamp
+  ```
+  select timestamp from signoz_logs.distributed_logs_v2 order by timestamp asc limit 1
+  ```
+  Incase you find empty response for the above query, please make sure that you upgraded to version `>=0.53` first.
+
+### For Kubernetes
+
+- Exec into your clickhouse container
+    ```bash
+    kubectl exec -n platform -it chi-my-release-clickhouse-cluster-0-0-0 -- sh
+
+    clickhouse client
+    ```
+* Get the starting timestamp 
+   ```
+  select timestamp from signoz_logs.distributed_logs order by timestamp asc limit 1
+  ```
+* Get the ending timestamp
+  ```
+  select timestamp from signoz_logs.distributed_logs_v2 order by timestamp asc limit 1
+  ```
+  Incase you find empty response for the above query, please make sure that you upgraded to version `>=0.53` first.
+
+
+Next we will run the migrator with arguments that will copy the data from the old to the new table.
+
+### For Docker
+
+```bash
+docker run --name signoz-migrate-55-data --network clickhouse-setup_default \
+  -it -d signoz/migrate:0.55-data -host=clickhouse -port=9000 -dest_host=clickhouse -dest_port=9000 -start_ts=<START_TIMESTAMP> -end_ts=<END_TIMESTAMP> -batch_size=40000 -batch_duration=5
+```
+
+Please replace `<START_TIMESTAMP>` and `<END_TIMESTAMP>`
+
+Here 
+* **start_ts** is the starting timestamp for the logs data to be migrated
+* **end_ts** is the ending timestamp for the logs data to be migrated
+* **batch_duration** is the window size of the data that will be fetched from the old table
+* **batch_size** is the size of the batches that will be written to the new table
+
+Steps to check logs:
+
+```bash
+docker logs -f signoz-migrate-55-data
+```
+
+In case of failure and you have to run the script again, 
+
+* Get the updated ending timestamp
+  ```
+  select timestamp from signoz_logs.distributed_logs_v2 order by timestamp asc limit 1
+  ```
+* Make sure to cleanup the container before running the migration script again. 
+  ```bash
+  docker stop signoz-migrate-55-data
+
+  docker rm signoz-migrate-55-data
+  ```
+
+
+### For Kubernetes
+
+* Create a `pod.yaml` file
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: signoz-migrate-data
+spec:
+  restartPolicy: Never
+  containers:
+    - name: signoz-migrate-data
+      image: signoz/migrate:0.55-data
+      args:
+        - "-host=my-release-clickhouse"
+        - "-dest_host=my-release-clickhouse"
+        - "-username=admin"
+        - "-dest_username=admin"
+        - "-password=<ADMIN_PASSWORD>"
+        - "-dest_password=<ADMIN_PASSWORD>"
+        - "-start_ts=<START_TIMESTAMP>"
+        - "-end_ts=<END_TIMESTAMP>"
+        - "-batch_size=40000"
+        - "-batch_duration=5"
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
+```
+
```

Comment:
include instructions to create pod using the yaml.

```suggestion

Now, create the `signoz-migrate-data` pod using the YAML:

```yaml
kubectl -n <NAMESPACE> apply -f pod.yaml
```

```